### PR TITLE
add button to request exact question

### PR DESCRIPTION
**Pull request by lcb931023**
The button is an anchor link to `https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`. This opens the form for adding a new question.

### CODE REVIEW

**Comment by Aprillion** [comment]
what was the issue with this requested, please? I don't remember the exact requirement 😅

**Comment by lcb931023** [comment]
This feature is to allow UI users submit new questions they have in mind, in case Stampy doesn't return a satisfying match.

@plexish mentioned that this is important to work on during weekly meetings in ~October. Though, I was unable to attend the last few ones, so maybe thing's changed :p

The need to allow new question submission was also referenced in https://github.com/StampyAI/stampy-ui/issues/37

**Comment by plexish** [comment]
Yup, this still seems like a key feature.

File: app/components/search.tsx [hunk 1]
```diff
@@ -106,6 +108,17 @@ export default function Search({
...
               />
             ))}
+          <a
+            href={`https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`}
+            target='_blank' rel="noreferrer"
+            className={`transparent-button result-item`}
+            style={{display: 'flex', alignItems: 'center', textDecoration: 'none'}}
+            key='none-of-the-above'
+            title='Request a new question'
+            onMouseDown={(e) => e.preventDefault()}
```
Comment (Aprillion) [inline app/components/search.tsx:118]: what's the purpose of this line, please?

File: app/components/search.tsx [hunk 2]
```diff
@@ -106,6 +108,17 @@ export default function Search({
                 }}
               />
             ))}
+          <a
+            href={`https://stampy.ai/wiki/Special:FormStart?form=Q&page_name=${searchInput}`}
+            target='_blank' rel="noreferrer"
```
Comment (Aprillion) [inline app/components/search.tsx:113]: could you please check `npm run lint` and/or enable eslint and prettier in your editor? JSX props should be in double quotes in our code style settings, I believe

File: app/components/search.tsx [hunk 3]
```diff
@@ -106,6 +108,17 @@ export default function Search({
...
+            className={`transparent-button result-item`}
+            style={{display: 'flex', alignItems: 'center', textDecoration: 'none'}}
```
Comment (Aprillion) [inline app/components/search.tsx:115]: I would prefer adding a className and corresponding style in the `css` file instead of non-dynamic inline styles

**Review by Aprillion — APPROVED** [review]
thanks for pointer to the issue, this looks like a good first step 👍

I have a few minor comments, please fix if it's clear what I'm asking or let me know if not clear and I can fix those at a later time after merging.

**Review by Aprillion — APPROVED** [review]
I updated this PR as per my previous comments and I also moved the link below the search results box to distinguish it more from search results:
image

@ccstan99 I did not consult your design for this is a temporary link to the wiki, the future solution for anonymous request of new questions directly from UI might look different..

**Pull request merged**